### ISSUE TITLE
LanguageServiceAccessor.getLSWrapper()

### ISSUE THREAD

**Issue by mniewrzal** [eclipselabs/eclipse-language-service#63]
Very often I have NullPointerException (see stacktrace). I don't know why and I'm not sure how to fix this. This particular stacktrace was thrown when I tried to select problem in Problems view. 

I'm testing this LSP client with [php-language-server](https://github.com/felixfbecker/php-language-server) and maybe I integrate something incorrectly.
```
!ENTRY org.eclipse.jface 4 2 2016-10-31 12:36:51.693
!MESSAGE Problems occurred when invoking code from plug-in: "org.eclipse.jface".
!STACK 0
java.lang.NullPointerException
	at org.eclipse.languageserver.LanguageServiceAccessor.getLSWrapper(LanguageServiceAccessor.java:201)
	at org.eclipse.languageserver.LanguageServiceAccessor.getLanguageServer(LanguageServiceAccessor.java:175)
	at org.eclipse.languageserver.operations.codeactions.LSPCodeActionMarkerResolution.hasResolutions(LSPCodeActionMarkerResolution.java:96)
	at org.eclipse.ui.internal.ide.registry.MarkerHelpRegistry.hasResolution(MarkerHelpRegistry.java:182)
	at org.eclipse.ui.internal.ide.registry.MarkerHelpRegistry.hasResolutions(MarkerHelpRegistry.java:151)
	at org.eclipse.ui.internal.views.markers.QuickFixPropertyTester.test(QuickFixPropertyTester.java:37)
```

**Comment by mickaelistria** [comment]
I think it's a "regular" bug, not an integration/configuration issue. You're using the HEAD from master?

**Comment by mniewrzal** [comment]
Yes + io.typefox.lsapi.\* (0.3.0.201609200806) bundled in target platform.

**Comment by mickaelistria** [comment]
I also face this issue regularly with OmniSharp. I've looked at the code and it seems to me the necessary guards are there... I don't understand yet how this can be fixed.

**Comment by mickaelistria** [comment]
Should be fixed with #74
